code/datums/riding.dm

```diff
@@ -249,6 +253,71 @@
 					buckled_mob.pixel_x = 10
 					buckled_mob.pixel_y = 5
 
+//prototype repair vehicle
+/datum/riding/space/repair
+	keytype = null
+	vehicle_move_delay = 1
+	var/obj/machinery/repair_turret/turret = null
+	var/obj/vehicle/space/speedbike/repair/R = null
```

Comment:
The ATV one is probably a leftover from me trying to figure out why it was bugged, your 2nd example isn't a dupe though. I need to give both vars the proper types or just using ridden is going to return errors because it doesn't know what ridden.turret is.